app/models/miq_action.rb

```diff
@@ -221,6 +222,11 @@ def action_audit(_action, rec, inputs)
                     :message      => "Policy #{msg}: policy: [#{inputs[:policy].description}], event: [#{inputs[:event].description}]")
   end
 
+  def action_run_ansible_playbook(action, rec, inputs)
+    service_template = ServiceTemplate.find(action.options[:service_template_id])
+    Api::ServiceTemplateWorkflow.create(service_template, {}).submit_request
```

Comment:
it's a lib/services/api/ helper class.  right now only used by Api and probably should stay that way. I wonder if that specific logic should be moved to the model and used by Api and from here. 

/cc @imtayadeway